MacPass/MPAutotypeDaemon.m

```diff
@@ -315,6 +309,18 @@ - (void)_performAutotypeForContext:(MPAutotypeContext *)context {
     /* TODO - we might be able to a notification to check if the app actally was activated instead of guessing a waiting time */
     usleep(1 * NSEC_PER_MSEC);
   }
+  
+  NSUserNotification *notification = [[NSUserNotification alloc] init];
```

Comment:
Moving the notification here will display it when the user selected the entry via the selection dialog. The intended behaviour was to not bother the user about this anymore since they already know what's going to happen.

Reply:
You are right, I was thinking, there will be a unified reminder for any match.